libs/sdk-ui-dashboard/package.json

```diff
@@ -36,7 +36,7 @@
     },
     "scripts": {
         "clean": "rm -rf ci dist esm coverage *.log styles/css && jest --clearCache",
-        "api-extractor": "mkdir -p api && [ -z \"${CI}\" ] && (api-extractor run -l) || (api-extractor run)",
+        "api-extractor": "mkdir -p api && [ -z \"${CI}\" ] && (api-extractor run -l; exit 0) || (api-extractor run)",
```

Comment:
What is the reason for this change?

Reply:
if api extractor run fails, it will otherwise run second time (due to exit 1 and ||)